### ISSUE TITLE
Add Radboud Affiliation Tag to CNGT glosses and very old original Signbank NGT glosses w/o creator

### ISSUE THREAD

**Issue by susanodd** [Signbank/Global-signbank#1211]
@uklomp refers to this in #1143 with high priority.

https://github.com/Signbank/Global-signbank/issues/1143#issue-2120799714

The Home page of Signbank mentions Corpus CNGT, ECHO Corpus, and Handy Signs, all included in the NGT dataset.
(#1214 to rewrite the Home page.)

The Radboud CNGT glosses can be identified by the Corpus/Speaker/Document/Frequency models.

- Add a virtual environment command to add the Radboud Affiliation Tag to all glosses with Frequency CNGT data. COMPLETE

**Comment by susanodd** [comment]
Allow Annotation and Lemma duplications when they are for glosses in the same dataset but in a different corpus.

Thus to avoid deletion of CNGT NGT glosses because they also appear in a UvA corpus for NGT.

The CNGT glosses have Gloss IDs linked to the EAF files of Corpus CNGT. These signs should not be deleted and should retain their Gloss ID.

NGT used to be synonymous with CNGT. What is the relation to the Dataset and the Corpus? The CNGT signs are public.

**Comment by susanodd** [comment]
UPDATE

The colors of the affiliation tags (generic name for this kind of tag) can be changed in the Admin.

The Radboud Red has been set for the Radboud tag. The default color is Green. (As requested in #1143 )

**Comment by susanodd** [comment]
@uklomp this is completed on signbank!

I added the affiliation to a few more UvA usernames (some had more than one signbank username).

Then I ran the command to add the user affiliation to glosses they had created (NGT, Oefen, tstMH).
If there was no creator (glosses prior to 2015/2016), then Radboud was added. (There were no other datasets at that time.)

The code also automatically adds the affiliation to new glosses.
(I deployed the code first, then was busy adding adding the affiliation to users I could determine, by searching for created by, and checking when it was created.)
Then ran the command.

There are likely some Radboud glosses that did not get tagged, since I'm not familiar with the users.
Prior to the Change Dataset permission, the user only needed to be in group Editor. So there are Radboud glosses created by Lianne last spring, but you don't see her in the Dataset permissions.

(If you come across more glosses that don't have an affiliation tag in NGT, or in Oefen, then likely a username has not been tagged.)
